### ISSUE TITLE
ProfileSpecToXmlPersistenceTest.java: testSaveProfileSpecWithSomeResources() contains an incorrect hard coded control date

### ISSUE THREAD

**Issue by ross-spencer** [digital-preservation/droid#80]
Inspecting this test failure on my system:

```
Distributor ID: Ubuntu
Description:    Ubuntu 14.04.3 LTS
Release:    14.04
Codename:   trusty
```

I notice the profile written to disk is:

```
<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
<Profile Id="untitled-1">
    <CreatedDate>2009-01-01T00:00:00+13:00</CreatedDate>
    <ProfileSpec>
        <Resources>
            <File>
```

But the hard coded control string is:

```
    DateTimeFormatter formatter = ISODateTimeFormat.dateTimeNoMillis();
    String control = "<Profile Id=\"untitled-1\">"
            + "  <CreatedDate>2009-01-01T00:00:00Z</CreatedDate>"
            + "  <ProfileSpec>"
            + "    <Resources>"
```

Note the: 2009-01-01T00:00:00Z

Would not see the time-zone included in the string as we see in the DROID generated string:  2009-01-01T00:00:00+13:00

Perhaps there is a way to suppress the time zone as I'd expect with UTC, or have the time zone difference generated on the fly at runtime, so UTC + zonal difference.

**Comment by snail1966** [comment]
Interesting, I get the same behaviour on Windows 7 when I set the Time Zone to Auckland/Wellington (UTC offset of 12 + 1 hour daylight saving at this time of year, hence the 13).  So it seems to be a java date issue rather than OS specific. Not clear how it's passing in Travis which I believe defaults to US settings.  May need  to move code involving dates to use a library such as joda.

**Comment by ross-spencer** [comment]
Also referenced here: https://groups.google.com/forum/#!topic/droid-list/6uDZ2ljJams

**Comment by rhubner** [comment]
Hi,
fixed in #163 2d96cbe

Radek
